.prh.yaml

```diff
@@ -0,0 +1,129 @@
+version: 1
+rules:
+  - pattern: vald
+    expected: Vald
+    options:
+      wordBoundary: true
+    specs:
+      - from: vald
+        to: Vald
+      - from: valdcli
+        to: valdcli
+  - pattern: helm
+    expected: Helm
+    options:
+      wordBoundary: true
+  - pattern:
+      - Golang
+      - golang
+      - /go(?! to)/
+    expected: Go
+    options:
+      wordBoundary: true
+    specs:
+      - from: go
+        to: Go
+      - from: going
+        to: going
+      - from: go to
+        to: go to
+      - from: golang
+        to: Go
+      - from: algorithm
+        to: algorithm
+  - pattern: python
+    expected: Python
+    options:
+      wordBoundary: true
+  - pattern: clojure
+    expected: Clojure
+    options:
+      wordBoundary: true
+  - pattern: java
+    expected: Java
+    options:
+      wordBoundary: true
+  - pattern:
+      - node.js
+      - nodejs
+    expected: Node.js
+  - pattern:
+      - Yahoo Japan!
+      - Yahoo! Japan
+      - YJ
+    expected: Yahoo Japan
+  - pattern: ngt
+    expected: NGT
+    options:
+      wordBoundary: true
+    specs:
+      - from: length
+        to: length
```

Comment:
What does mean? :thinking:

Reply:
same as https://github.com/vdaas/vald/pull/1715/files#r911559600 🤔

Reply:
@vankichi @kevindiu 
thank you for comment 🙏 
same as https://github.com/vdaas/vald/pull/1715/files#r911565093
in this case, we can delete specs field.

Reply:
same as https://github.com/vdaas/vald/pull/1715/files/ad3c703447715d0beb8df6062386d8fde528a20a#r911571438